.github/workflows/docker-image.yml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     branches:
     - main
+    - automatic-build
   workflow_dispatch:
 
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: low-code-ros
```

Comment:
Maybe a bit more detailed image name? Afaik low-code-ros was the headline for both projects.  Maybe sqe22-robitic-welding-backend?

Reply:
I would rather opt for keeping "sqe" or other module related stuff out of the image name. In case this project survives the end of this semester "sqe" will sound a little puzzling for developers and users.

I suggest `low-code-welding-backend`.